adafruit_sht4x.py

```diff
@@ -46,7 +52,7 @@ class CV:
     """struct helper"""
 
     @classmethod
-    def add_values(cls, value_tuples):
+    def add_values(cls, value_tuples) -> None:
```

Comment:
See https://github.com/adafruit/Adafruit_CircuitPython_TLA202x/blob/main/adafruit_tla202x/__init__.py for help annotating `CV.add_values()`, as they work identically.